src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int SIZE_OF_ARRAY = 6;
```

Comment:
this is not an informative constant name - size of what array? 

it would be better to name it like `NUMBER_OF_FIGURES`